assets/post_install_macOS.sh

```diff
@@ -33,26 +33,28 @@ osascript \
 # Use Intel packages if the Python binary is x84_64, i.e. not native Apple Silicon
 # (This also applies to an Intel binary running on Apple Silicon through Rosetta)
 # https://conda-forge.org/docs/user/tipsandtricks.html#installing-apple-intel-packages-on-apple-silicon
-export PYTHON_PLATFORM=`${DSTROOT}/.mne-python/bin/conda run python -c "import platform; print(platform.machine())"`
+DSTBIN=${DSTROOT}/.mne-python/bin
+PYTHON_PLATFORM=$(${DSTBIN}/conda run python -c "import platform; print(platform.machine())")
```

Comment:
Is there a specific reason why you're not using backticks instead of `$(...)`? I find the backticks easier to read

Reply:
Yes, some BASH linter (in vscode I think?) complained that backticks were bad and that I should use `$()` instead